lib/agent0/agent0/utilities/predict.py

```diff
@@ -0,0 +1,234 @@
+"""Predict the outcome of trades."""
+
+from __future__ import annotations
+
+import logging
+from copy import deepcopy
+from typing import NamedTuple
+
+from ethpy.hyperdrive.interface.read_interface import HyperdriveReadInterface
+from ethpy.hyperdrive.state import PoolState
+from fixedpointmath import FixedPoint
+
+# need to cover a wide variety of cases
+# pylint: disable=too-many-arguments
+# being very explicit
+# pylint: disable=too-many-locals
+
+YEAR_IN_SECONDS = 31_536_000
+BLOCKS_IN_YEAR = YEAR_IN_SECONDS / 12
```

Comment:
I think you either want

`SECONDS_IN_YEAR` & `BLOCKS_IN_YEAR` to mean "the number of X (seconds or blocks) in Y (a year)"

OR

`YEAR_IN_SECONS` & `YEAR_IN_BOCKS` to mean "a year, but in units of X (seconds or blocks)"

You currently mix & match the two

Reply:
changed to YEAR_IN_BLOCKS